src/main/webapp/js/common.js

```diff
@@ -750,3 +750,17 @@ if (!String.prototype.includes) {
         return String.prototype.indexOf.apply(this, arguments) !== -1;
     }
 }
+
+/**
+ * Check if the input value is blank
+ * (Used in validating comments given by instructors)
+ * 
+ * @param str
+ * @returns boolean
+ */
```

Comment:
- Don't mention anything about the method's usage in the header comment
- The `@returns` can be made clearer (e.g true if ..., false if ...)
- "Checks if the input value is a blank string" might be more accurate since it also encapsulates the function's behavior on non-String input